homeassistant/components/netgear/device_tracker.py

```diff
@@ -153,21 +142,19 @@ def _update_info(self):
 
         Returns boolean if scanning successful.
         """
-        if not self.success_init:
-            return
-
-        _LOGGER.info("Scanning")
+        _LOGGER.debug("Scanning")
 
         results = self.get_attached_devices()
 
+        _LOGGER.debug("Scan result: \n%s", pformat(results))
```

Comment:
Let's guard this logging statement to make sure that we're going to log it. Otherwise the call to `pformat` is wasteful and those can be expensive. 

https://github.com/NabuCasa/hass-nabucasa/blob/dev/hass_nabucasa/iot_base.py#L90-L91